docs/src/pages/components/data-grid/sorting/BasicExampleDataGrid.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import { DataGrid } from '@mui/x-data-grid';
+import { useDemoData } from '@mui/x-data-grid-generator';
+
+const VISIBLE_FIELDS = ['name', 'rating', 'country', 'dateCreated', 'isAdmin'];
+
+export default function BasicExampleDataGridPro() {
```

Comment:
```suggestion
export default function BasicExampleDataGrid() {
```

Reply:
I'm making a PR for that using https://www.npmjs.com/package/eslint-plugin-filenames

Reply:
@alexfauquette This project is abandoned: https://github.com/import-js/eslint-plugin-import/issues/662#issuecomment-667963894

There's also https://github.com/minseoksuh/eslint-plugin-consistent-default-export-name but the rule we need is so simple that it might be better to create one in https://github.com/mui-org/material-ui-x/blob/master/packages/eslint-plugin-material-ui/src/rules/no-direct-state-access.js than to have to manage another dependency.